modules/account-lib/src/coin/sol/transactionBuilderFactory.ts

```diff
@@ -0,0 +1,42 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BaseTransactionBuilderFactory } from '../baseCoin';
+import { NotImplementedError } from '../baseCoin/errors';
+import { TransferBuilder } from './transferBuilder';
+import { WalletInitializationBuilder } from './walletInitializationBuilder';
+import { TransactionBuilder } from './transactionBuilder';
+import { Transaction } from './transaction';
+
+export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  /** @inheritdoc */
+  from(raw: Uint8Array | string): TransactionBuilder {
+    throw new NotImplementedError('from not implemented');
+  }
+
+  /** @inheritdoc */
+  getWalletInitializationBuilder(tx?: Transaction): WalletInitializationBuilder {
+    return TransactionBuilderFactory.initializeBuilder(tx, new WalletInitializationBuilder(this._coinConfig));
+  }
+
+  /** @inheritdoc */
+  getTransferBuilder(tx?: Transaction): TransferBuilder {
+    return TransactionBuilderFactory.initializeBuilder(tx, new TransferBuilder(this._coinConfig));
+  }
+
```

Comment:
Method left:
-getBuilder

Reply:
Not necessary, the method from does this.